common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
Oh, I see that main_requirejs.js is really only for tests. There should be someplace else to put that.